starter-code/iteration2/src/app/app.component.html

```diff
@@ -0,0 +1,7 @@
+<!--The content below is only a placeholder and can be replaced.-->
+<div style="text-align:center">
+  <h1>
+    Learn how to code with Ironhack!
+  </h1>
+</div>
```

Comment:
Since you have created a `main-component` you will have a new selector with the name `app-main-component`. 
This one you have t put it in the html where you want to use it. For instance, in this exercise we want to use it here, on `app.component.html`. So you must put here a line with `<app-main-component></app-main-component>` for showing this component here.